### PR TITLE
tidb-retool-integration

### PR DESCRIPTION
**Pull request by Chenhao-Huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-retool-integration.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
+1. Get TiDB Details & Whitelist Retool IP Address
+
+    You can get your database connection details from whoever set up the database. You'll also need to get them to [whitelist][1] the Retool IP address.
```
Comment (qiancai) [inline tidb-retool-integration.md:4]: What does "TiDB Details" mean? Would you please provide the specific information that is needed?

Comment (qiancai) [inline tidb-retool-integration.md:4]:
```suggestion
    You can get your database connection details from whoever set up the database. You'll also need to get them to [whitelist][https://docs.retool.com/docs/connect-database-resource#connecting-your-database] the Retool IP address.
```

File: tidb-retool-integration.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+2. Add TiDB as a resource in Retool
+
+    Create a new resource in Retool, and select "MySQL" as the type.
+    Enter your database connection details.
+    * Name: &lt;Your resource name&gt;
```
Comment (TomShawn) [inline tidb-retool-integration.md:10]: Avoid using HTML escape characters.

File: tidb-retool-integration.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
```
Comment (TomShawn) [inline tidb-retool-integration.md:1]: If this document is only applicable to TiDB Cloud, please place the file in the `tidb-cloud` directory.

File: tidb-retool-integration.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+1. Get TiDB Details & Whitelist Retool IP Address
```
Comment (TomShawn) [inline tidb-retool-integration.md:2]: Could you first explain what Retool is? What users can do with it? The scenario?

File: tidb-retool-integration.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+    You can get your database connection details from whoever set up the database. You'll also need to get them to [whitelist][1] the Retool IP address.
+
+2. Add TiDB as a resource in Retool
+
+    Create a new resource in Retool, and select "MySQL" as the type.
+    Enter your database connection details.
```
Comment (TomShawn) [inline tidb-retool-integration.md:9]: Please avoid manual line breaks.

File: tidb-retool-integration.md (hunk 3 above)
Comment (TomShawn) [inline tidb-retool-integration.md:1]: Each document should have a metadata section. Please add one according to the following template.
```
---
title: xx
summary: xx
---
```

**Comment by Chenhao-Huang** [comment]
All comments are addressed. PTAL

**Comment by Chenhao-Huang** [comment]
All comments are addressed. PTAL

**Pull request closed without merging**